static/js/components/forms/CheckoutForm.js

```diff
@@ -250,7 +261,9 @@ export class InnerCheckoutForm extends React.Component<InnerProps, InnerState> {
                     Apply
                   </button>
                 </div>
-                {formatErrors(errors.coupons)}
+                {this.isPromoCodeApplied
```

Comment:
This will not work. You need to call the function

```suggestion
                {this.isPromoCodeApplied()
```

Reply:
this is working, I had tested the code and verified now as well. But changing it on your request.

Reply:
It works for displaying the success message. It does not work for displaying the error message. `this.isPromoCodeApplied` returns `true` if that property is not null or undefined. Since that function is defined, it always evaluates to `true`, so the false case (the error message) would never be executed